Dockerfile.centos8

```diff
@@ -0,0 +1,32 @@
+FROM centos:8
+ENV HOME /
+RUN dnf update -y
+RUN dnf install -y epel-release
+RUN dnf module install -y perl:5.30
```

Comment:
I'm missing perl... 😿

Reply:
🙀
I can't reproduce this in my environment.
Does the error occur even with the latest Docker image?

Reply:
It works in my environment. 🤔